app/test_main.py

```diff
@@ -1,3 +1,47 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_years",
+        [
+            pytest.param(
+                0, 0, [0, 0],
+                id="return two zeroes when both ages are zero"
+            ),
+            pytest.param(
+                -100, -100, [0, 0],
+                id="return two zeroes when both ages are negative"
+            ),
+            pytest.param(
+                28, 28, [3, 2],
+                id="first occurrence where numbers should differ"
+            ),
+            pytest.param(
+                14, 14, [0, 0],
+                id="14 cat/dog years should convert into 0 human age."
+            ),
+            pytest.param(
+                1256, 1256, [310, 248],
+                id="should work properly with big numbers"
+            ),
+            pytest.param(
+                25.2, 25.2, [2.0, 2.0],
+                id="should work properly with float values"
+            )
+        ]
+    )
```

Comment:
You should add more tests for all age boundaries, for example, you check here that 14, 14 gives 0, 0 in the result, but you don't check that 15, 15 gives 1, 1 in the result, etc.

![image](https://user-images.githubusercontent.com/104218550/220347861-584c75a3-693a-4f77-b498-688726e488a6.png)